website_portal_purchase/README.rst

```diff
@@ -0,0 +1,64 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+============================
+Website Portal for Purchases
+============================
+
+This module adds the partner's purchase documents in the frontend portal.
+Suppliers will be able to see the list and state of their request for
+quotations, purchase orders and supplier invoices.
+
+Usage
+=====
+
+To use this module, you need to:
+
+1. Assign auth user group to the user.
+2. If you install 'portal_sale' you can invite user to access it.
+3. User can access to his request quotations, orders and supplier invoices
+from his account in user account link.
```

Comment:
In .rst files, these lines must be aligned with their parent, so you need to put 2 paces before this line.